### PR TITLE
Use flickity to display panoramas

### PR DESCRIPTION
**Pull request by WillNilges**
TIL that a component called a carousel exists and there are libraries for it. This uses one of said libraries, "Flickity" to (hopefully) make the panoramas display a little better.

### CODE REVIEW

**Comment by WillNilges** [comment]
image
image
image

File: src/meshdb/settings.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "db.mesh",
     "db.nycmesh.net",
     "127.0.0.1",
+    "10.0.2.2",
```
Comment (Andrew-Dickinson) [inline src/meshdb/settings.py:44]: Why do we want to allow this?

**Review by Andrew-Dickinson — APPROVED** [review]

File: src/meshapi/templates/widgets/panorama_viewer.html [hunk 2]
```diff
@@ -1,5 +1,15 @@
-<div style='display:flex; flex-direction: row; width:100%; overflow-x:auto; max-height:300px; padding-left: 10px;'>
-	{% for panorama in widget.value %}
-		<a href="{{ panorama }}" target="_blank"><img src="{{ panorama }}" style="margin-right:10px"/></a>
-	{% endfor %}
+<script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js"></script>
```
Comment (james-otten) [inline src/meshapi/templates/widgets/panorama_viewer.html:1]: Can this safely use SRI?

File: src/meshdb/settings.py (hunk 1 above)
Comment (WillNilges) [reply src/meshdb/settings.py:44]: oops that's an artifact of my testing on windows.

File: src/meshapi/templates/widgets/panorama_viewer.html (hunk 2 above)
Comment (james-otten) [inline src/meshapi/templates/widgets/panorama_viewer.html:1]:
```suggestion
<script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js" integrity="sha384-6ma6pGRLjeu6PkX+yOaR0dUGt6OQl9evbv3H2kdLWwIXtUztaQrWUlbRBPVpDH4B" crossorigin="anonymous"></script>
```

Comment (WillNilges) [reply src/meshapi/templates/widgets/panorama_viewer.html:1]: I'm not sure. I could probably serve a copy of flickity from our server if that would be better.

**Pull request merged**